gpg-agent-placeholder

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# prevent starting real gpg-agent locally if it's redirected via split-gpg2
+if [ -e /run/qubes-service/split-gpg2-client ]; then
+    exit 0
+fi
+# otherwise, launch gpg-agent
+exec "$(gpgconf --list-components | grep ^gpg-agent: | cut -d ':' -f 3)" "$@"
```

Comment:
In that case, I suggest using `-o pipefail`, `-o/dev/stdout` (to catch `-ENOMEM`), and a separate command so that `gpgconf` failing results in the `exec` not happening.

Reply:
> `-o/dev/stdout` (to catch `-ENOMEM`)

if `gpg-agent` line is missing, grep will fail.